arcgis-ios-sdk-samples/Features/Filter by definition expression or display filter/DefinitionExpressionDisplayFilterViewController.swift

```diff
@@ -0,0 +1,108 @@
+// Copyright 2016 Esri.
```

Comment:
Should this be updated to 2022?

Reply:
Agreed. Although this sample is derived from an existing one, I feel it has changed enough to be a new sample.